### ISSUE TITLE
Cannot read impersonated service account credentials

### ISSUE THREAD

**Issue by vertuli** [abdolence/sbt-gcs-resolver#5]
Uh oh, the plugin is failing when attempting to read impersonated service account credentials.

To repeat this, impersonate a valid service account on your local machine:
```
gcloud auth application-default login --impersonate-service-account=example_account@example_project.iam.gserviceaccount.com
```

And then run any sbt action (e.g. `sbt test`). 

```
[error] Unable to find/initialise google credentials: java.lang.ClassCastException: scala.collection.convert.Wrappers$IterableWrapper cannot be cast to java.util.List. Publishing/resolving artifacts from GCP is disabled.
```

**Comment by abdolence** [comment]
Interesting behaviour :) let me check it

**Comment by abdolence** [comment]
I think I fixed this with v1.6.1 and it worked for me on my test project. Let me know if this doesn't work for you for some reason and thanks for the report!
